fields/types/localfiles/LocalFilesType.js

```diff
@@ -170,7 +171,7 @@ localfiles.prototype.addToSchema = function() {
 		delete: function(element_id) {
 			if (exists(this, element_id)) {
 				var values = this.get(field.path);
-				var value = values.find(function (val) { return val._id === element_id; });
+				var value = values.find(function (val) { return val._id == element_id; });
```

Comment:
If you use the === you are comparing the also the type of data.
Here, one is a string, and the other is an ObjectId ?
The condition will always fail in admin using the type comparator ===.